### ISSUE TITLE
Soft handset audio playback with headphones plugged in

### ISSUE THREAD

**Issue by overkillthemighty** [teamacid/android_device_samsung_galaxys4gmtd#22]
I noticed that while the headphone jack is plugged in the phone plays audio thru the jack as well as the handset speaker (the one for calls)

**Comment by maxfierke** [comment]
It's really soft though, right? If so, I've noticed that as well and will mark this bug as confirmed.

**Comment by overkillthemighty** [comment]
Yeah. I noticed it when I turned my car off and I noticed the music was
still playing softly.
On Jun 26, 2012 4:10 PM, "Max Fierke" <
reply@reply.github.com>
wrote:

**Comment by maxfierke** [comment]
That's the exact same scenario as me. Issue confirmed.

We'll look into it.
